source/index.html

```diff
@@ -21,8 +21,11 @@
         <span class="author">Penelope Piper</span>
       </li>
     </ul>
-
-    <button id="new_comment_button">New Comment</button>
+    <form id = "new-comment">Comments:</br>
+      <textarea id="comment-body" value="comment here" rows="4" cols="50" name="comment" form="new-comment"></textarea></br>
```

Comment:
it's not necessary to add an id specifically to these two inputs. you can use a different selector to select this element - see these [docs](https://api.jquery.com/attribute-equals-selector/)